avm/res/network/virtual-network/main.json

```diff
@@ -6,7 +6,7 @@
     "_generator": {
       "name": "bicep",
       "version": "0.29.47.4906",
-      "templateHash": "17645744687566805489"
+      "templateHash": "15637892116799403398"
```

Comment:
I still see the template hash changes. Could this please be reverted too? The readme update should not impact this

Reply:
done